client/src/components/pages/RespondentQuiz.js

```diff
@@ -7,27 +7,31 @@ import {
 	getRespondentQuiz,
 	takeQuiz,
 } from "../../context/respondent/RespondentState";
+import AlertTestSubmit from '../Alerts/AlertTestSubmit';
 
 const RespondentQuiz = () => {
     const [respondentState, respondentDispatch] = useRespondent();
     const { error, respondent, quiz_resp } = respondentState;
-    const { hashKey, quizId } = useParams();
+    const { iv, hashKey, quizId } = useParams();
     const navigate = useNavigate();
 
     const [questionsAnswered, setQuestionsAnswered] = useState([]);
+    const [alert, setShowAlert] = useState(false);
+    const [confirmMessage, setConfirmMessage] = useState('');
+    const [confirm, setConfirm] = useState(false);
 
     let time = quiz_resp && quiz_resp.timeLimit;
-    const [stateTimeLimit, setStateTimeLimit] = useState(0);
+    const [stateTimeLimit, setStateTimeLimit] = useState(1);
 
 	const onTimeLimitChange = useCallback(() => {
 		setInterval(
-			() => setStateTimeLimit((stateTimeLimit) => stateTimeLimit - 1),
+			() => setStateTimeLimit((stateTimeLimit) => stateTimeLimit - 34),
```

Comment:
do you mind explaining what this means?

Reply:
Oops this needs to be reverted back to just - 1. I did that for some of the tests for quicker results and I forgot to change it back. Let me push the edit